packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -1,21 +1,27 @@
 // @ts-check
 
 import { assert, details as X } from '@agoric/assert';
-import { makeWeakStore as makeNonVOWeakStore } from '@agoric/store';
+import {
+  makeWeakStore as makeNonVOWeakStore,
+  makeStore as makeNonVOStore,
+} from '@agoric/store';
 import { E } from '@agoric/eventual-send';
 import { AmountMath } from '@agoric/ertp';
 import { Far } from '@agoric/marshal';
 
 import { isOfferSafe } from './offerSafety';
 import { assertRightsConserved } from './rightsConservation';
+import { addToAllocation, subtractFromAllocation } from './allocationMath';
 
 /** @type {CreateSeatManager} */
 export const createSeatManager = (zoeInstanceAdmin, getAssetKindByBrand) => {
```

Comment:
There is no type `SeatManager` or `SeatKit`. But from the comment on `CreateSeatManager` I would guess that the record returned at the bottom is the SeatManager. If so, then per our previous discussion, I do not understand why this is `createSeatManager` rather than `makeSeatManager`.

Reply:
This is unchanged in this PR, so I won't address it here, but I think this is fine, since seatManager is an abstract concept only used internally. Since there is no `seatManager` reified as an object, I think `create` is fine as well.